src/main/java/com/Model/User.java

```diff
@@ -0,0 +1,68 @@
+package com.Model;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    private String emailAddress;
+    private String password;
```

Comment:
you still have a password here that looks like it's in the clear... maybe document here what measures you're taking so that it's encrypted?

Reply:
All user passwords are encrypted before saving into the repository. 
Check UserService class save(User user, HttpServletRequest request) method. 
I will add Java document here.